### PR TITLE
[stable/postgresql] Add the possibility to explicitly select namespaces from which Ingress traffic could be accepted.

### PR DESCRIPTION
**Pull request by machine424**
#### What this PR does / why we need it:
When networkPolicy is enabled ( and allowExternal is set to false) only Pods that are in the networkPolicy's namespace and that match other criteria, the ones that can be selected by one of the podSelectors, can reach the DB. But sometimes, we want the DB to be accessible from other namespaces, in this case, we can use the `Values.networkPolicy.explicitNamespacesSelector` LabelSelector to select these namespaces, note that the networkPolicy's should also be explicitly added.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

@desaintmartin

### CODE REVIEW

File: stable/postgresql/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql
-version: 7.6.0
+version: 7.6.1
```
Comment (desaintmartin) [inline stable/postgresql/Chart.yaml]: Please bump minor version since it's new feature

**Review by desaintmartin — CHANGES_REQUESTED** [review]

**Comment by desaintmartin** [comment]
Thanks!
/ok-to-test

File: stable/postgresql/Chart.yaml (hunk 1 above)
Comment (javsalgar) [reply stable/postgresql/Chart.yaml]: +1

**Review by javsalgar — COMMENTED** [review]
Thanks for the PR. Please bump the minor version and after that it looks good to me

File: stable/postgresql/Chart.yaml (hunk 1 above)
Comment (machine424) [reply stable/postgresql/Chart.yaml]: Done

**Pull request merged**